terragrunt/aws/base/vpc.tf

```
@@ -1,8 +1,8 @@
 module "vpc" {
-  source = "github.com/cds-snc/terraform-modules//vpc?ref=v8.0.0"
+  source = "github.com/cds-snc/terraform-modules//vpc?ref=v10.3.0"
   name   = "${var.product_name}-${var.tool_name}"
 
-  high_availability  = true
+  availability_zones = 3
```

Can you also try setting the CIDR subnet bits to see if this prevents the destroy/recreate:
```suggestion
  availability_zones = 3
  cidrsubnet_newbits = 8
```

---

Thanks for the suggestion Pat!